.gitignore

```diff
@@ -68,7 +68,7 @@ node_modules/
 .sass-cache
 
 # Celery
-celerybeat-schedule
+celerybeat-schedule.*
```

Comment:
Why this `*` here?

Reply:
`git` was tracking the files such as `celerybeat-schedule.bak`, `celerybeat-schedule.dat` etc. So to un-track these files, I have put them into `.gitignore`.